tasks/main.yml

```diff
@@ -1,8 +1,6 @@
 ---
-# tasks file for ansible-role-lustre_client
-
 - include_tasks: redhat.yml
-  when: lustre_client_enabled and ansible_os_family == "RedHat"
```

Comment:
Functionality should be left here. See details in the comment for  defaults/main.yml.